packages/backend-core/src/sql/sql.ts

```diff
@@ -170,6 +170,8 @@ function convertBooleans(query: SqlQuery | SqlQuery[]): SqlQuery | SqlQuery[] {
   return query
 }
 
+const COMPLEX_ID_OPERATOR = "_complexIdOperator"
```

Comment:
Could we move this perhaps into type (I know its not strictly a type, but we put enums and other stuff in there as well, we probably would want to use it for typing in the future as well

Reply:
You mean in @budibase/types? We could also add strict types if they live in backend core, as this is where the only "usage" lives

Reply:
Amended!